src/Serilog.Sinks.Datadog.Logs/LoggerConfigurationDatadogLogsExtensions.cs

```diff
@@ -20,12 +20,18 @@ public static class LoggerConfigurationDatadogExtensions
         /// </summary>
         /// <param name="loggerConfiguration">The logger configuration.</param>
         /// <param name="apiKey">Your Datadog API key.</param>
+        /// <param name="source">The integration name.</param>
+        /// <param name="service">The service name.</param>
+        /// <param name="tags">Custom tags.</param>
         /// <param name="configuration">The Datadog logs client configuration.</param>
         /// <returns>Logger configuration</returns>
         /// <exception cref="ArgumentNullException">A required parameter is null.</exception>
         public static LoggerConfiguration DatadogLogs(
```

Comment:
I'd like to see "host" as a settable attribute here as well that flows into a DataDogMessage.Host property.